resolver/resolver.go

```diff
@@ -3357,20 +3390,6 @@ func (r *Resolver) resolveCELValue(ctx *context, env *cel.Env, value *CELValue)
 	if issues.Err() != nil {
 		return errors.Join(append(r.celRegistry.errors(), issues.Err())...)
 	}
-	if len(r.celRegistry.usedEnumValueMap) != 0 {
```

Comment:
It seems we don't need the `usedEnumValueMap` field to `CELRegistry`?

Reply:
That's correct ! However, since we would like to eventually implement a proper mechanism for handling enum value, I plan to keep the mechanism in place.